src/service/data_products/taxa_count.py

```diff
@@ -44,6 +44,24 @@
 
 _MAX_COUNT = 20  # max number of taxa count records to return
 
+# TaxaCount fields. These need to match the field names in the TaxaCount class below
+_FLD_TAXA_COUNT_MATCH_COUNT = "match_count"
+_FLD_TAXA_COUNT_SEL_COUNT = "sel_count"
+
+# The default sorting order for taxa counts results
+_DEFAULT_SORT_ORDER = [names.FLD_TAXA_COUNT_COUNT,
+                       _FLD_TAXA_COUNT_MATCH_COUNT,
+                       _FLD_TAXA_COUNT_SEL_COUNT]
```

Comment:
Technically this is just `list(_SORT_PRIORITY_VALUE_MAP.values()).reverse()`

Reply:
👍

Reply:
:+1: